src/test/java/de/tum/in/www1/artemis/notification/NotificationFactory.java

```diff
@@ -29,6 +44,13 @@ public static GroupNotification generateGroupNotification(ZonedDateTime notifica
         return groupNotification;
     }
 
+    /**
+     * Generates a SystemNotification with the given arguments.
+     *
+     * @param notificationDate The notification date of the SystemNotification
+     * @param expireDate       The expiry date of the SystemNotification
```

Comment:
parameter is named expireDate (wrong, it should be either expiryDate or expirationDate) and explanation expiry date (correct in british english)

Reply:
Implemented feedback
Please note that changing attribute and method names in SystemNotification.java is out of scope for this PR